modelci/app/v1/endpoints/model.py

```diff
@@ -6,43 +6,32 @@
 Date: 6/20/2020
 """
 import asyncio
+import http
 import json
 import shutil
 from pathlib import Path
 from typing import List
 
 from fastapi import APIRouter, File, UploadFile, Depends
-from fastapi.exceptions import RequestValidationError
+from fastapi.exceptions import RequestValidationError, HTTPException
 from pydantic.error_wrappers import ErrorWrapper
 from starlette.responses import JSONResponse
 
 from modelci.hub.manager import register_model
-from modelci.persistence.service import ModelService
-from modelci.persistence.service_ import get_by_id
-from modelci.types.bo import Framework, Engine, Task
-from modelci.types.models import MLModel, BaseMLModel
-from modelci.types.vo.model_vo import ModelListOut, Framework as Framework_, Engine as Engine_, \
-    Task as Task_
+from modelci.persistence.service_ import get_by_id, get_models, update_model, delete_model, exists_by_id
+from modelci.types.models import MLModel, BaseMLModel, ModelUpdate, Framework, Engine, Task
 
 router = APIRouter()
 
 
-@router.get('/', response_model=List[ModelListOut])
-def get_all_model(architecture: str = None, framework: Framework_ = None, engine: Engine_ = None, task: Task_ = None,
+@router.get('/', response_model=List[MLModel], response_model_by_alias=False)
+def get_all_models(architecture: str = None, framework: Framework = None, engine: Engine = None, task: Task = None,
                   version: int = None):
-    if framework is not None:
-        framework = Framework[framework.value.upper()]
-    if engine is not None:
-        engine = Engine[engine.value.upper()]
-    if task is not None:
-        engine = Task[task.value.upper()]
+    models = get_models(architecture=architecture, framework=framework, engine=engine, task=task, version=version)
+    return models
 
-    models = ModelService.get_models(architecture=architecture, framework=framework, engine=engine, task=task,
-                                     version=version)
-    return list(map(ModelListOut.from_bo, models))
 
-
-@router.get('/{id}')
+@router.get('/{id}', response_model=MLModel, response_model_by_alias=False)
```

Comment:
Will this work? As I am manually converting the response to JSON, we don't need response model.